continuity.jmeter/src/main/java/org/continuity/jmeter/controllers/TestPlanController.java

```diff
@@ -39,15 +42,22 @@
 	 * @return A bundle holding the test plan or a 404 error response if not found.
 	 */
 	@RequestMapping(value = GET, method = RequestMethod.GET)
-	public ResponseEntity<JMeterTestPlanBundle> getTestPlan(@PathVariable String id) {
+	public ResponseEntity<String> getTestPlan(@PathVariable String id) {
```

Comment:
Do you call this method directly from Java? If not, the former implementation should do exactly the same as your implementation. You can always deserialize this on caller-side as a String.

Reply:
The method is called from Java. See in project "Orchestrator" - class "LoadTestController" - method "getLoadTest". A JsonNode is expected but I can only provide a String, and a String in a JsonNode results in a String without line breaks but with "\n".

Reply:
Ok, it's called from Java, but not as method call but as REST request. Hence, it still should work to return the JMeterTestPlanBundle.
In which format is the BenchFlow test you return? I guess it's not really only a string...